scripts/mkcloud

```diff
@@ -1047,7 +1049,9 @@ Optional
     upgrade_cloudsource='' (default='')
         set new cloudsource for upgrade process
     TESTHEAD='' | 1  (default='')
-        use Media from Devel:Cloud:Staging and add test update repositories (except for GM* targets)
+        use Media from Devel:Cloud:Staging (except for GM* targets)
+    want_test_updates='' | 1  (default='')
+        add test update repositories
```

Comment:
documentation should go into onadmin_help
since the variable is not used in mkcloud itself